From 3758e4c642d60c445ed58b7dce094f24f3cdafc6 Mon Sep 17 00:00:00 2001
From: Girish Sharma <girish.sharma@einfochips.com>
Date: Tue, 1 Dec 2015 11:04:31 +0530
Subject: [PATCH 24/40] defconfig: Remove unwanted machine types

Following machines were removed from msm8960_defconfig
Since they create large kernel image
CONFIG_MACH_MSM8960
CONFIG_MACH_MSM8960_MTP
CONFIG_MACH_MSM8960_FLUID
CONFIG_MACH_MSM8960_LIQUID
CONFIG_MACH_MSM8930_CDP
CONFIG_MACH_MSM8930_MTP
CONFIG_MACH_MSM8930_FLUID
CONFIG_MACH_MSM8930_EVT
CONFIG_MACH_MSM8627_CDP
CONFIG_MACH_MSM8627_MTP
CONFIG_MACH_APQ8064_MTP
CONFIG_MACH_MPQ8064_CDP
CONFIG_MACH_MPQ8064_HRD
CONFIG_MACH_MPQ8064_DTV

Only below machines are enabled
CONFIG_MACH_APQ8064_DB600C=y
CONFIG_MACH_APQ8064_SIM=y
CONFIG_MACH_APQ8064_RUMI3=y
CONFIG_MACH_APQ8064_CDP=y
CONFIG_MACH_APQ8064_LIQUID=y

Change-Id: Ibe5757347cea2891f12b55f3117ec97a644df8a9
Signed-off-by: eInfochips Team <ei-qc8064@einfochips.com>
---
 arch/arm/configs/msm8960_defconfig | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/arch/arm/configs/msm8960_defconfig b/arch/arm/configs/msm8960_defconfig
index fcd08d9..cec0c12 100644
--- a/arch/arm/configs/msm8960_defconfig
+++ b/arch/arm/configs/msm8960_defconfig
@@ -43,25 +43,25 @@ CONFIG_ARCH_APQ8064=y
 CONFIG_MSM_KRAIT_TBB_ABORT_HANDLER=y
 CONFIG_MACH_MSM8960_SIM=y
 CONFIG_MACH_MSM8960_RUMI3=y
-CONFIG_MACH_MSM8960_CDP=y
+# CONFIG_MACH_MSM8960_CDP is not set
 CONFIG_MACH_APQ8064_DB600C=y
-CONFIG_MACH_MSM8960_MTP=y
-CONFIG_MACH_MSM8960_FLUID=y
-CONFIG_MACH_MSM8960_LIQUID=y
-CONFIG_MACH_MSM8930_CDP=y
-CONFIG_MACH_MSM8930_MTP=y
-CONFIG_MACH_MSM8930_FLUID=y
-CONFIG_MACH_MSM8930_EVT=y
-CONFIG_MACH_MSM8627_CDP=y
-CONFIG_MACH_MSM8627_MTP=y
+# CONFIG_MACH_MSM8960_MTP is not set
+# CONFIG_MACH_MSM8960_FLUID is not set
+# CONFIG_MACH_MSM8960_LIQUID is not set
+# CONFIG_MACH_MSM8930_CDP is not set
+# CONFIG_MACH_MSM8930_MTP is not set
+# CONFIG_MACH_MSM8930_FLUID is not set
+# CONFIG_MACH_MSM8930_EVT is not set
+# CONFIG_MACH_MSM8627_CDP is not set
+# CONFIG_MACH_MSM8627_MTP is not set
 CONFIG_MACH_APQ8064_SIM=y
 CONFIG_MACH_APQ8064_RUMI3=y
 CONFIG_MACH_APQ8064_CDP=y
-CONFIG_MACH_APQ8064_MTP=y
+# CONFIG_MACH_APQ8064_MTP is not set
 CONFIG_MACH_APQ8064_LIQUID=y
-CONFIG_MACH_MPQ8064_CDP=y
-CONFIG_MACH_MPQ8064_HRD=y
-CONFIG_MACH_MPQ8064_DTV=y
+# CONFIG_MACH_MPQ8064_CDP is not set
+# CONFIG_MACH_MPQ8064_HRD is not set
+# CONFIG_MACH_MPQ8064_DTV is not set
 # CONFIG_MSM_STACKED_MEMORY is not set
 CONFIG_KERNEL_MSM_CONTIG_MEM_REGION=y
 # CONFIG_MSM_FIQ_SUPPORT is not set
-- 
1.9.1

